Can't build on macOS 10.15

(PR in flight)

```
-- Found SFML 2.5.1 in /Users/nicolemazzuca/src/sfml/build
-- Configuring done
CMake Error in CMakeLists.txt:
  Imported target "CoreFoundation" includes non-existent path

    "/System/Library/Frameworks/CoreFoundation.framework/Headers"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "CoreFoundation" includes non-existent path

    "/System/Library/Frameworks/CoreFoundation.framework/Headers"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "CoreFoundation" includes non-existent path

    "/System/Library/Frameworks/CoreFoundation.framework/Headers"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
make: *** [cmake_check_build_system] Error 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -107,16 +107,8 @@ if( WIN32 )
 	set( LIB_PATH "lib" )
 elseif( APPLE )
 	find_library( COREFOUNDATION_LIBRARY CoreFoundation )
-	mark_as_advanced( COREFOUNDATION_LIBRARY )
 
-	add_library( CoreFoundation SHARED IMPORTED )
-	set_target_properties(
-		CoreFoundation PROPERTIES
-			IMPORTED_LOCATION "${COREFOUNDATION_LIBRARY}"
-			INTERFACE_INCLUDE_DIRECTORIES "/System/Library/Frameworks/CoreFoundation.framework/Headers"
-	)
-	
-	target_link_libraries( ${TARGET} PUBLIC CoreFoundation )
+	target_link_libraries( ${TARGET} PUBLIC ${COREFOUNDATION_LIBRARY} )
 	set( SHARE_PATH "${CMAKE_INSTALL_PREFIX}/share/SFGUI" )
 	set( LIB_PATH "lib" )
 elseif( "${CMAKE_SYSTEM_NAME}" MATCHES "Linux" )

